src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceEnums.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+import java.util.HashMap;
+
+public class SalesForceEnums {
+
+    public enum Theme {
+        CLASSIC("classic"),
+        LIGHT("lightning");
+
+        private String themeName;
+
+        private Theme(String themeName) {
+            this.themeName = themeName;
+        }
+
+        public String getThemeName() {
+            return themeName;
+        }
+
+        private static final HashMap<String, Theme> THEME_HASH_MAP = new HashMap<>();
+
+        static {
+            for (Theme theme : Theme.values()) {
+                THEME_HASH_MAP.put(theme.getThemeName(), theme);
+            }
+        }
+
+//        public static Theme getThemeEnumByThemeName(String themeName) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)